docs/guides/qunasys-quri-chemistry.ipynb

```diff
@@ -79,11 +79,12 @@
     "\n",
     "Detailed description of each argument is summarized in following table.\n",
     "\n",
-    "| Name       | Type | Description                        | Required | Default                             | Example           |\n",
-    "| :--------- | :--- | :--------------------------------- | :------- | :---------------------------------- | :---------------- |\n",
-    "| method     | str  | The algorithm name                 | Yes      | -                                   | “QSCI” or \"OPT_QSCI\"           |\n",
-    "| parameters | json | Details settings of the algorithm. | Yes      | -                                   |                   |\n",
-    "| backend_name    | str  | The backend name you want to use   | No       | The least busy one available        | “ibm_torino”      |"
+    "| Name         | Type | Description                        | Required | Default                          | Example              |\n",
+    "| :----------- | :--- | :--------------------------------- | :------- | :------------------------------- | :------------------- |\n",
+    "| method       | str  | The algorithm name                 | Yes      | -                                | “QSCI” or \"OPT_QSCI\" |\n",
+    "| parameters   | json | Details settings of the algorithm. | Yes      | -                                |                      |\n",
+    "| backend_name | str  | The backend name you want to use   | No       | The least busy one available     | “ibm_torino”         |\n",
+    "| instance     | str  | The instance you want to use       | No       | The default one you save locally |                      |"
```

Comment:
>  The default one you save locally

Maybe update to the following:

> [Selected consistent with Qiskit Runtime pattern](https://docs.quantum.ibm.com/guides/instances#specify-an-instance-in-your-code)

Reply:
I have updated it and added a link to it.